src/views/Profile/AdminTable/styles/AdminDatePicker.styles.ts

```diff
@@ -13,3 +13,12 @@ export const useStyles = makeStyles((theme: Theme) => ({
     },
   },
 }));
+
+export const useStylesForDatePicker = makeStyles({
+  datePicker: {
+    '& .MuiPickersDay-daySelected': {
```

Comment:
btw btw, I see `disableHighlightToday` for [docs on](https://mui.com/x/api/date-pickers/date-picker/#DatePicker-prop-disableHighlightToday) newer version of material-ui. And looking for this property over legacy 3.3.10 I see [some usage](https://github.com/search?q=repo%3Amui%2Fmaterial-ui-pickers%20disableHighlightToday&type=code). May it be that it worked also in 3.3.10?

Reply:
I've checked in codesandbox and surprisingly it does not allow me to add `@material-ui/pickers@3.3.10` like it if was unsupported. So I'm asking you to check this on Dokazovi,

Reply:
I've already checked it, and it doesn't allow me to add it too.

Reply:
We can't add "disableHighlightToday", because we don't have this property in our library version. I find it in node modules.